src/hotspot/share/opto/phaseX.cpp

```diff
@@ -1160,6 +1184,813 @@ bool PhaseIterGVN::verify_node_value(Node* n) {
   tty->cr();
   return true;
 }
+
+// Check that all Ideal optimizations that could be done were done.
```

Comment:
```suggestion
// Check that all Ideal optimizations that could be done were done.
// Returns true if it found missed optimization opportunities and false otherwise and for exceptions.
```
The return value was not immediately clear to me.